src/Controller/ControllerManager.php

```diff
@@ -1,15 +1,21 @@
 <?php
 
+declare(strict_types=1);
+
 namespace Laminas\Mvc\Controller;
 
-use Interop\Container\ContainerInterface;
+use interop\container\containerinterface;
```

Comment:
An error that is constantly repeated.

Reply:
I can't remember which rule causes this problem, but IIRC you need to append the import with `// phpcs:ignore` rather than disabling it in `phpcs.xml`